addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml

```diff
@@ -4,7 +4,55 @@
         xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
         xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
-    <!-- Supported Fritz! devices and features -->
+    <!-- Supported FRITZ! devices and features -->
+    <thing-type id="Comet_DECT">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="fritzbox" />
+        </supported-bridge-type-refs>
+        <label>Comet DECT</label>
+        <description>Comet DECT heating thermostat</description>
+        <channels>
+            <channel id="temperature" typeId="temperature" />
+            <channel id="actual_temp" typeId="actual_temp" />
+            <channel id="set_temp" typeId="set_temp" />
+            <channel id="eco_temp" typeId="eco_temp" />
+            <channel id="comfort_temp" typeId="comfort_temp" />
+            <channel id="next_change" typeId="next_change" />
+            <channel id="next_temp" typeId="next_temp" />
+            <channel id="battery_low" typeId="battery_low" />
```

Comment:
Please use the existing system channel type `system.low-battery` (see [here](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels)) instead of defining your own `battery_low` type.